tests/unit/test_errorhandler.py

```diff
@@ -43,13 +44,14 @@ def _assert_rc_and_error_message(case, error_handler):
         raise case.exception
     except BaseException as e:
         cr = error_handler.handle_exception(e, stdout, stderr)
-        nose.tools.eq_(cr, case.rc, case.exception.__class__)
-        nose.tools.assert_in(case.stderr, stderr.getvalue())
-        nose.tools.eq_(case.stdout, stdout.getvalue())
+        assert cr == case.rc, case.exception.__class__
+        assert case.stderr in stderr.getvalue()
+        assert case.stdout in stdout.getvalue()
```

Comment:
It looks like this is a v2 specific change. We moved the last assertion from `equals` to `in`. That actually looks like the correct fix, but wanted to double check this was intentional.

Reply:
Good catch. This should be equals.